elements/rh-alert/docs/10-style.md

```diff
@@ -1,88 +1,127 @@
-There are two alert variants available for use.
+{% section %}
+## Style
+
+An alert contains title text with an icon, body text, and a close button. They may also include action buttons below the text or inline links. There are two variants, toast and inline, which serve different purposes.
+
+### Anatomy
 
 {% example palette="light",
-           width=538,
-           alt="Alert component blueprint",
-           src="../alert-blueprint.svg" %}
+           width=498,
+           alt="Alert with numbers pointing to parts of the element",
+           src="../alert-anatomy.svg" %}
+
+<ol class="anatomy-labels">
+        <li>Severity indicator</li>
+        <li>Severity icon</li>
+        <li>Title</li>
+        <li>Body</li>
+        <li>Actions</li>
+        <li>Close button</li>
+</ol>
```

Comment:
```suggestion
1. Severity indicator
1. Severity icon
1. Title
1. Body
1. Actions
1. Close button
{.anatomy-labels}
```

Reply:
markdown counts for you
1. even though
1. this comment
1. only contains
1. `1.`

`{.anatomy-labels}` applies the class to the element it follows. Putting it on a new line makes it apply to the `ol`, instead of the the last `li`